docs/src/tutorials/aws-example-policy.json

```diff
@@ -0,0 +1,50 @@
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Action": "iot:Connect",
+      "Resource": "arn:aws:iot:eu-west-1:027519331087:client/${iot:ClientId}"
+    },
+    {
+      "Effect": "Allow",
+      "Action": "iot:Publish",
+      "Resource": "arn:aws:iot:eu-west-1:027519331087:topic/thinedge/devices/${iot:ClientId}/messages/*"
+    },
+    {
+      "Effect": "Allow",
+      "Action": "iot:Subscribe",
+      "Resource": "arn:aws:iot:eu-west-1:027519331087:topicfilter/thinedge/devices/${iot:ClientId}/messages/devicebound/*"
+    },
+    {
+      "Effect": "Allow",
+      "Action": "iot:Receive",
+      "Resource": "arn:aws:iot:eu-west-1:027519331087:topic/thinedge/devices/${iot:ClientId}/messages/devicebound/*"
+    },
+    {
+      "Effect": "Allow",
+      "Action": "iot:RetainPublish",
+      "Resource": "arn:aws:iot:eu-west-1:027519331087:topic/tedge/health/*"
+    },
+    {
+      "Effect": "Allow",
+      "Action": "iot:Publish",
+      "Resource": "arn:aws:iot:eu-west-1:027519331087:topic/tedge/health/*"
```

Comment:
What are these 2 topics? They seem not be bridged.

Reply:
This is the topic used for health notifications. Only a single topic here however one rule is for publish and the other is for retain publish.